src/core/qgsapplication.cpp

```diff
@@ -491,29 +491,29 @@ void QgsApplication::installTranslators()
   // Remove translators if any are already installed
   if ( mQgisTranslator )
   {
-    removeTranslator( mQgisTranslator );
-    delete mQgisTranslator;
-    mQgisTranslator = nullptr;
+    removeTranslator( mQgisTranslator.get() );
+    mQgisTranslator.reset( );
```

Comment:
```suggestion
    removeTranslator( mQgisTranslator.release() );
```

Reply:
Actually, we cannot do that because mQgisTranslator needs to be deleted and neither of release() or removeTranslator() would delete it.

Reply:
I missed that. Probably a cleaner approach would that removeTranslator() takes a std::unique_ptr, and we would move it into it

Reply:
this is a static Qt function

Reply:
ok, all settled then :-)